static/js/components/LearnerSearch.js

```diff
@@ -102,6 +103,7 @@ export default class LearnerSearch extends SearchkitComponent {
                 </Cell>
```

Comment:
this is totally not part of this PR, but 10 lines above this, the "Email Selected" button is given `id="download csv"`. think that's pretty obviously a typo. since youre already in this code right now, would you mind changing that as part of this PR or some other learner search front end PR?